agent/battle/data/chaldea_converter.py

```
@@ -0,0 +1,155 @@
+
+
+def _is_order_change_mystic_code(mystic_code_id: Optional[int]) -> bool:
+    """是否是换人礼装。"""
+    return (
+        isinstance(mystic_code_id, int)
+        and mystic_code_id in _ORDER_CHANGE_MYSTIC_CODE_IDS
+    )
+
+
+def _parse_target(options: Optional[dict]) -> Optional[int]:
+    """从 options 解析技能目标（playerTarget, 0-based -> 1-based）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)